src/main/java/org/folio/rest/impl/PatronBlockLimitsAPI.java

```diff
@@ -0,0 +1,118 @@
+
+public class PatronBlockLimitsAPI implements PatronBlockLimits {
+
+  private static final String PATRON_BLOCK_LIMITS = "patron_block_limits";
+  private static final String MAX_OUTSTANDING_FEEFINE_BALANCE_ID = "cf7a0d5f-a327-4ca1-aa9e-dc55ec006b8a";
+
+  @Validate
+  @Override
+  public void getPatronBlockLimits(int offset, int limit, String query, String lang,
+    Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
+    Context vertxContext) {
+
+    PgUtil.get(PATRON_BLOCK_LIMITS, PatronBlockLimit.class,
+      org.folio.rest.jaxrs.model.PatronBlockLimits.class, query, offset, limit,
+      okapiHeaders, vertxContext, GetPatronBlockLimitsResponse.class, asyncResultHandler);
+  }
+
+  @Validate
+  @Override
+  public void postPatronBlockLimits(String lang, PatronBlockLimit entity,
+    Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
+    Context vertxContext) {
+
+    Errors errors = isEntityValid(entity);
+    if (errors != null) {
+      asyncResultHandler.handle(succeededFuture(PostPatronBlockLimitsResponse
+          .respond422WithApplicationJson(errors)));
+      return;
+    }
+
+    PgUtil.post(PATRON_BLOCK_LIMITS, entity, okapiHeaders, vertxContext,
+      PostPatronBlockLimitsResponse.class, asyncResultHandler);
+  }
+
+  @Validate
+  @Override
+  public void putPatronBlockLimitsByPatronBlockLimitId(String patronBlockLimitId,
+    String lang, PatronBlockLimit entity, Map<String, String> okapiHeaders,
+    Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+
+    Errors errors = isEntityValid(entity);
+    if (errors != null) {
+      asyncResultHandler.handle(succeededFuture(PutPatronBlockLimitsByPatronBlockLimitIdResponse
+          .respond422WithApplicationJson(errors)));
+      return;
+    }
+
+    PgUtil.put(PATRON_BLOCK_LIMITS, entity, patronBlockLimitId, okapiHeaders,
+      vertxContext, PutPatronBlockLimitsByPatronBlockLimitIdResponse.class, asyncResultHandler);
+  }
+
+  @Validate
+  @Override
+  public void getPatronBlockLimitsByPatronBlockLimitId(String patronBlockLimitId,
+    String lang, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
+    Context vertxContext) {
+
+    PgUtil.getById(PATRON_BLOCK_LIMITS, PatronBlockLimit.class, patronBlockLimitId,
+      okapiHeaders, vertxContext, GetPatronBlockLimitsByPatronBlockLimitIdResponse.class,
+      asyncResultHandler);
+  }
+
+  @Validate
+  @Override
+  public void deletePatronBlockLimitsByPatronBlockLimitId(String patronBlockLimitId,
+    String lang, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
+    Context vertxContext) {
+
+    PgUtil.deleteById(PATRON_BLOCK_LIMITS, patronBlockLimitId, okapiHeaders,
+      vertxContext, DeletePatronBlockLimitsByPatronBlockLimitIdResponse.class,
+      asyncResultHandler);
+  }
+
+  private Errors isEntityValid(PatronBlockLimit entity) {
```

Comment:
Personally, if a method's name starts with `is...`, I expect it to return a boolean. How about renaming it to something like `validateEntity`?

Reply:
It was done with the same code style as in another modules, for example: `org.folio.rest.impl.FixedDueDateSchedulesAPI#postFixedDueDateScheduleStorageFixedDueDateSchedules` (mod-circulation-storage)
Also from my perspective the method named as `validateEntity` should return void type or throw an exception, but it our case it returns Errors

Reply:
I agree with Oleksandr.
The name in mod-circulation-storage is misleading, too, don't copy that naming scheme.
The method name should be `validateEntity` or even `validate` because `validate...` does not indicate the return type.

Reply:
There already are two methods that start with `validate` and return `Errors`:
`Errors validateRangeForDoubleValueType`
`Errors validateRangeForIntegerValueType`

Reply:
OK, renamed.
I have renamed the similar method in PatronBlockConditionsAPI as well.